lib/tbd/geo.rb

```diff
@@ -306,26 +326,41 @@ def properties(model = nil, surface = nil)
       end
     end
 
-    surf[:conditioned] = true
+    unless argh.key?(:setpoints)
+      heat = heatingTemperatureSetpoints?(model)
+      cool = coolingTemperatureSetpoints?(model)
+      argh[:setpoints] = heat || cool
+    end
+
+    if argh[:setpoints]
+      surf[:heating] = setpts[:heating] unless setpts[:heating].nil?
+      surf[:cooling] = setpts[:cooling] unless setpts[:cooling].nil?
+    else
+      surf[:heating] = 21.0
+      surf[:cooling] = 24.0
+    end
+
+    surf[:conditioned] = surf.key?(:heating) || surf.key?(:cooling)
```

Comment:
Moved from psi.rb. The logic is revised (and simplified) to better account for _indirectly-conditioned_ spaces (not just _plenums_). The User Guide will need to reflect these final changes once merged.